src/components/errorAnimation/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Lottie from 'react-lottie';
+import * as animationData from '../../assets/404-error.json';
+import { Container } from './styles';
+
+const ErrorAnimation = () => {
```

Comment:
Acho que um nome mais específico fica melhor aqui para não confundir com outras animações de erro.
```suggestion
const NotFoundErrorAnimation = () => {
```